src/sass/base/_typography.scss

```diff
@@ -71,11 +71,6 @@
     }
 }
 
-.section-subtitle {
-    font-size: 3.3rem;
-    margin-bottom: 2%;
-}
-
```

Comment:
I removed this class from typography - I realized while working today that I was thinking about the typography file incorrectly. 

Instead of the one source of truth for type-related styles - which becomes unmanageable and hard to maintain - its the sour of truth for GLOBAL type-related styles. Keep it simple, stupid. I will have to slowly fix this error, moving component specific type-styling to their appropriate sass partials.